Lib/test/test_asyncio/test_locks.py

```diff
@@ -1000,5 +1000,31 @@ def test_release_no_waiters(self):
         self.assertFalse(sem.locked())
 
 
+class LockLoopTests:
+    def setUp(self):
```

Comment:
```suggestion
    """Tests the deprecation of creating asyncio objects outside of a running
    event loop.
    """ 
    
    def setUp(self):
```

The docstring is optional, but I think the formatting fix is likely going to be required to be PEP8 compliant (https://www.python.org/dev/peps/pep-0008/#blank-lines).

Reply:
thanks! I made the changes